onnx/defs/tensor/defs.cc

```diff
@@ -721,7 +723,13 @@ ONNX_OPERATOR_SET_SCHEMA(
             "If the tensor is not evenly splittable the last chunk will be smaller.",
             AttributeProto::INT,
             false)
-        .SetDoc(Split_ver18_doc)
+        .Attr(
+            "mode",
+            "Uneven split mode. "
+            "Possible values are 'torch' (default) and 'legacy'.",
+            AttributeProto::STRING,
+            std::string("torch"))
```

Comment:
Should we keep the legacy as default?

Reply:
As far as I know torch is the desired one, and I set legacy to be default in version converter when coming from opset 19 to 20. I can change it.

Reply:
Good question. I think the current choice is ok. Especially if it is only the torch exporter that uses this, and presumably the torch converter needs the new option, not the old one.